src/Services/privateApiService.js

```diff
@@ -52,8 +52,18 @@ const privateRequestPut = async(path,id,body) => {
 
     return data;
 };
-
-
+export const privateDelete = async(url,id) =>{
+    const getAuthorization = getAuthorizationHeader();
+    if(!getAuthorization.Authorization) return;
+    else{
+        return await axios.delete(url,id,{
+            headers:getAuthorization,
+        })
+        .then((response) => (response.data))
+          .catch((error) => console.log(error));
+    }
+}
+export default Get;
```

Comment:
this export default get is not necessary because the const are being exported at the end of the file, you should delete this export default get line and also the export from the privateDelete, then you should add it into the last line export:
```
export {privatePost, Get, privateRequestPut,getAuthorizationHeader,privateGet, privateDelete }
```